plugins/extractors/mysql/mysql.go

```diff
@@ -64,22 +57,17 @@ func (e *Extractor) Info() plugins.Info {
 	}
 }
 
-// Validate validates the configuration of the extractor
-func (e *Extractor) Validate(configMap map[string]interface{}) (err error) {
-	return utils.BuildConfig(configMap, &Config{})
-}
-
 // Init initializes the extractor
 func (e *Extractor) Init(ctx context.Context, configMap map[string]interface{}) (err error) {
-	if err = utils.BuildConfig(configMap, &e.config); err != nil {
+	if err = utils.BuildConfig(configMap, &e.Config); err != nil {
 		return plugins.InvalidConfigError{}
 	}
 
 	// build excluded database list
-	e.buildExcludedDBs()
+	e.ExcludedDbs = extutils.BuildExcludedDBs(defaultDBList)
```

Comment:
ExcludeDBs is not common to all extractors, should let each extractors handle it themselves? wdyt?

Reply:
Do you mean that not every data source has a default list of DB that needs to excluded?

Cause I thought if there are some default dblist for each source, then maybe we can keep the array with their names within each extractor seperately but the BaseExtractor can have it build in util package.

Reply:
yes, for example, `kafka`, `bigquery`, `metabase`, `tableau`.

Reply:
oh okay, I thought we allowed custom ExcludeDBList from users at one point, which lets users avoid metadata extraction from a particular set of topic/DBs in the source. But if there are no things at this point then I'll move it back to extractors.